packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -59,22 +65,36 @@ const MediaReplaceFlow = ( {
 		}
 	};
 
+	const onError = ( message ) => {
+		const errorElement = document.createElement( 'div' );
+		errorElement.innerHTML = renderToString( message );
+		const renderMsg =
+			errorElement.textContent || errorElement.innerText || '';
+		setTimeout( () => {
```

Comment:
Would be great to add a comment about why the timeout is needed.

Reply:
Correct, I've fiddled so much with this that I forgot it ain't obvious ;)